### PR TITLE
Dev5

### PR DESCRIPTION
**Pull request by YanaShcherba**

### CODE REVIEW

File: src/components/List.jsx [hunk 1]
```diff
@@ -1,43 +1,53 @@
...
-            return (
-              <li key={data.id}><img src={data.image}/>
-                <p>Name: {data.name}.</p>
-                <p>Phone Number: {data.number}</p>
-                <button key={data.id} className="delete-item" onClick={() => this.props.setDeletion(this.props.initData)}>
-                  Delete Contact
-                </button>
+      <div className="list-container">
+        <input
+          styleName="form-control"
```
Comment (maksimshenets) [inline src/components/List.jsx]: styleName? It cause an error

File: src/components/List.jsx [hunk 2]
```diff
@@ -1,43 +1,53 @@
...
+          placeholder="Who are you looking for?"
+          onChange={this.filterHadler}
+        />
+        <button className="add"><Link to="add"> New </Link></button>
+        <ul className="list">
+          {this.props.contactList
+            .filter(list => list.name.indexOf(this.props.visibilityFilter) !== -1)
+            .map((list) => (
+              <li key={list.id}><img src={list.image} alt="" />
```
Comment (maksimshenets) [inline src/components/List.jsx:29]: image

File: src/reducers/listReducer.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      number: '+375299090900',
+    }, {
+      id: 4,
+      image: 'http://lorempixel.com/400/200/cats',
+      name: 'Siel Phantomhive',
+      number: '+375266666900',
+    },
+  ],
+  visibilityFilter: '',
+  edit: {
```
Comment (maksimshenets) [inline src/reducers/listReducer.jsx]: Why we need this field in store?

File: src/components/List.jsx (hunk 1 above)
Comment (YanaShcherba) [reply src/components/List.jsx]: whoops.i just tried it,nothing good happened,and i forgot to remove it. there was className as well

Comment (maksimshenets) [reply src/components/List.jsx]: ok, work in the same pull please

Comment (YanaShcherba) [reply src/components/List.jsx]: yey,figured out what was the problem)

Comment (YanaShcherba) [reply src/components/List.jsx]: everything was fixed (i hope).

**Comment by maksimshenets** [comment]
Looks like everything works correct. @YanaShcherba resolve conflicts and merge.

**Comment by maksimshenets** [comment]
BTW, why cats so angry?=)

**Comment by YanaShcherba** [comment]
probably someone on lorempixel has some kind of problems with cats)

**Pull request merged**